android/app/src/staging/AndroidManifest.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+    <application
+            android:networkSecurityConfig="@xml/network_security_config">
```

Comment:
If later we find that the traffic/network isn't returning proper text, we can add `android:usesCleartextTraffic="true"` in here.